plugins/main/public/controllers/management/components/management/configuration/cluster/cluster.js

```diff
@@ -58,10 +85,22 @@ class WzCluster extends Component {
     let mainSettingsConfig = {
       ...currentConfig['com-cluster'],
       disabled:
-        currentConfig['com-cluster'].disabled === 'yes'
-          ? 'disabled'
-          : 'enabled',
+        currentConfig['com-cluster'].disabled === true ? 'disabled' : 'enabled',
     };
+
+    if (currentConfig['com-cluster'].haproxy_helper) {
+      mainSettingsConfig = {
+        ...mainSettingsConfig,
+        haproxy_helper: {
+          ...currentConfig['com-cluster'].haproxy_helper,
+          haproxy_disabled:
+            currentConfig['com-cluster'].haproxy_helper.haproxy_disabled ===
+            true
```

Comment:
**nitpick**: is the `haproxy_disabled` coming as a boolean? It seems to be configured as a string with `yes`/`no`

Reply:
It seems in 4.7.1 is a boolean, so this fixes an unnoticed bug.